runtime/src/accounts_db.rs

```diff
@@ -275,7 +279,9 @@ pub struct AccountsDB {
     file_size: u64,
 
     /// Thread pool used for par_iter
-    thread_pool: ThreadPool,
+    pub thread_pool: ThreadPool,
```

Comment:
is this right?  I'd think you want this inside something with a lock around it, as opposed to being shared among all threads that might scan()...